### PR TITLE
project phase one

### PR DESCRIPTION
**Pull request by olidi10**
Hi Mary,unable to upload image

### CODE REVIEW

**Comment by olidi10** [comment]
Hope you got my project.
   Thanks

File: index.html [hunk 1]
```diff
@@ -1,13 +1,49 @@
...
-    <meta charset="utf-8" />
+
+    <title>My Favorite Things</title>
     <link rel="stylesheet" href="style.css" />
+
 </head>
+
 <body>
+    <h1 id="header">Welcome Class</h1>
+    <div id="container">
```
Comment (mbMosman) [inline index.html]: The favorites HTML should be built in the JavaScript using jQuery.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        $("#container").show();
+        $("h2").hide();
+        var myfavorites = "Things I like best."
+        var newItem = "Good food nice Restaurant"
+        $("h1").html(myfavorites);
+        $("section.conclude").text(newItem);
+
+        //hide all the items and fade them in one at a time.
+        $("li").hide().each(function (i) {
+            $(this).delay(500 * i).fadeIn(1800);
```
Comment (mbMosman) [inline script.js]: This borrows from the grocery/to-do list example, but isn't what was asked for in this project.  You should build the HTML here, not just hide it and then fade it in.

**Comment by mbMosman** [comment]
Make sure to pay attention to the requirements, and ask questions if you're not sure what it is that you need to do.

File: script.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        });
+
+    });
+
+    //Create a variable,change text and adding a click event to display List of interest. 
+    $(":header").on("click", function () {
+        $("#container").show();
+        $("h2").hide();
+        var myfavorites = "Things I like best."
+        var newItem = "Good food nice Restaurant"
```
Comment (mbMosman) [inline script.js]: You should always define the variables at the top of their scope (at the start of a function).

**Pull request closed without merging**